libs/ui-toolkit/src/components/lozenge/lozenge.tsx

```diff
@@ -10,7 +10,7 @@ interface LozengeProps {
 }
 
 const getLozengeClasses = (
-  variant: LozengeProps['variant'],
+  variant?: LozengeProps['variant'],
```

Comment:
non-blocking: optional params can have bad influence for test coverage calculation - a new branch is formed, that demands his test case

Reply:
fair point, there were no tests for checking css classes for this component. not sure what's our aim for coverage, and what is worth testing with these generic, super simple components, the presence of class names seems very restrictive and also not very useful in most cases. nevertheless, added a case for the missing intent variant to storybook, so we at least have it there.